packages/eslint-plugin/src/rules/jsx-pragma/index.ts

```diff
@@ -62,10 +65,16 @@ const findReactDeclarationWithDefaultImport = (
 };
 
 function createFixer(context: Rule.RuleContext, source: SourceCode, options: Options) {
-  const compiledImports = findLibraryImportDeclarations(context, [COMPILED_IMPORT]);
+  const compiledImports = [
+    // TODO: Should this just be one call, eg. `[COMPILED_IMPORT, ...options.importSources]`?
```

Comment:
is this a TODO for us? 😅

Reply:
Nope, I forgot this.  Fixed in 5534012e

Doing `findLibraryImportDeclarations(context, [COMPILED_IMPORT, ...options.importSources])` instead of this spread and join should be more performant than two separate calls.  I'm not quite too concerned about performance enough to cache this across the main ESLint context and this `createFixer` context, but best to consolidate better.

Reply:
imo `findLibraryImportDeclarations(context, [COMPILED_IMPORT, ...options.importSources])` is less confusing yeah


i don't like the person below who made the second parameter of `findLibraryImportDeclarations` optional, that confused me a lot
![Screenshot 2024-09-11 at 11 24 10](https://github.com/user-attachments/assets/65445351-0340-4a9b-a467-1f293202dd87)